ci/jenkins/jobs/projects-lab.yaml

```diff
@@ -1010,6 +1067,63 @@
                       default-excludes: true
                       fingerprint: false
                       only-if-success: false
+      - '{name}-{test_name}-for-pull-request':
```

Comment:
We may not need three jobs for ovs-containerd validation, I think only the e2e test is sufficient because conformance and networkpolicy should not have differences between userspace OVS and containerized OVS.

Reply:
Removed containerd-ovs jobs for conformance and network policy tests